syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -604,6 +604,9 @@ def pow(self, power):
          - Divide power by 2 and multiply base to itself (if the power is even)
          - Decrement power by 1 to make it even and then follow the first step
         """
+        if power < 0:
+            raise RuntimeError("Integers to negative integer powers are not allowed.")
```

Comment:
```suggestion
            raise RuntimeError("Negative integer powers are not allowed.")
```
Claiming the base of an exponent to be an integer might not be ideal (although it is that way in an encrypted AST) to a user who might have encrypted mayebe a float.